src/tennissimulator/menu/Menu.java

```diff
@@ -100,61 +107,55 @@ private Gender resolveGender(String gender) {
 
     private int promptUserForRating(Organization organization) {
         System.out.print("Enter a " + organization.toString() + " rating: ");
+
         return scanner.nextInt();
     }
 
     private String promptUserForName() {
         System.out.print("Enter a full player name: ");
+
         return scanner.nextLine();
     }
 
     private String promptUserForGender() {
         System.out.print("Enter a gender (man/woman): ");
+
         return scanner.nextLine();
     }
 
-    private void selectPlayersForMatchSimulation() {
-        playersDatabase.print();
-        if (playersDatabase.getPlayers().isEmpty()) {
-            return;
-        }
+    private String promptUserForId() {
+        System.out.println("\nChoose player ID: ");
 
-        Player firstPlayer = resolvePlayer(promptUserForName());
-        if (!isValidPlayer(firstPlayer)) {
-            return;
-        }
+        return scanner.nextLine();
+    }
 
-        Player secondPlayer = resolvePlayer(promptUserForName());
-        if (!isValidPlayer(secondPlayer)) {
+    private void selectPlayersForMatchSimulation() throws RuntimeException {
+        playersDatabase.print();
+        if (playersDatabase.getPlayers().isEmpty()) {
             return;
```

Comment:
Looks like a place for introduction of another exception for a case when database is empty.

Reply:
Introduced `EmptyDatabaseException`.